planet/clients/subscriptions.py

```diff
@@ -76,19 +71,20 @@ async def create_subscription(self, request: dict) -> dict:
             ClientError
 
         """
-        # Temporary marker for behavior of module with unpatched state.
-        if _fake_subs is None:
-            raise NotImplementedError
+        try:
+            missing_keys = {'name', 'delivery', 'source'} - request.keys()
+            if missing_keys:
+                raise RuntimeError(
```

Comment:
In the other clients, usage errors result in a ClientError. Let's keep that pattern in this client as well.

Reply:
ah, got it.